src/lister_haxe/lister_syscmd.hx

```diff
@@ -0,0 +1,25 @@
+import Sys.*;
+
+class Lister_SysCmd {
```

Comment:
This class is here solely to be a wrapper for the interpreter to call.  Ideally we'll be able to compile `Lister` into whatever.  Since this class imports `Sys`, it can only target "system" platforms (Java, C#, neko, etc.)